src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link } from 'react-router-dom';
+import { userLogOut } from '../../util/loginUser';
+import './navbar.css';
+
+export interface Iprops {
+  userInfo: { id: number; name: string };
+}
+const Navbar = ({ userInfo }: Iprops) => {

```

Comment:
Files should be named after their components. I rarely use index files other than exporting other things. What happens is you end up with tons of index files in your editor and aren't able to tell which one is for which component.

Reply:
Oh i see the problem, ok i will do it like that!